.vib/spring-cloud-dataflow/cypress/cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
I think we don't need to add this file to the repo and leave it for local testing only.